packages/framework-provider-aws/src/library/read-models-adapter.ts

```diff
@@ -59,7 +62,7 @@ export async function storeReadModel(
         },
       })
       .promise()
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Since when this is possible? In theory TS syntax doesn't allow type annotations in `catch` clauses